### PR TITLE
add messages for some common connection errors

### PR DESCRIPTION
**Pull request by leogermani**
Adds error messages to some common XML-RPC connection errors.

These error messages will be displayed in the Jetpack Debugger.

#### Jetpack product discussion

p9dueE-1Ce-p2

#### Does this pull request change what data or activity we track or use?

No

##### Setting up:

* Start with a fresh Jetpack site, connected and authorized
* Install / activate the Debug Helper plugin (already available on docker env) - https://wp.me/p9dueE-1Ce
* Go to Jetpack >Dev tools and activate the Broken Token module
* set the `JETPACK_DEV_DEBUG` constant to `true`
* Go to Jetpack > Broken token
* Click "Store these options"

##### Test 1:

* Click "Set invalid blog token"
* Visit the Jetpack debugger
* Back to your dashboard, go to Jetpack > XML-RPC Errors
* Confirm there's an error with error code `no_valid_token` and message `Invalid blog token`.
* Clear all errors
* go back to Jetpack > broken token and click "Restore from stored options"

##### Test 2:

* Click "Set invalid user tokens"
* Visit the Jetpack debugger
* Back to your dashboard, go to Jetpack > XML-RPC Errors
* Confirm there's an error with error code `no_valid_token` and message `Invalid token for user XX` (where XX is the user ID).
* Clear all errors
* go back to Jetpack > broken token and click "Restore from stored options"

##### Test 3:

* Click "Clear user tokens"
* Visit the Jetpack debugger
* Back to your dashboard, go to Jetpack > XML-RPC Errors
* Confirm there's an error with error code `no_user_tokens` and message `No user tokens found` 
* Clear all errors
* go back to Jetpack > broken token and click "Restore from stored options"

##### Test 4:

* Click "Clear blog token"
* Visit the Jetpack debugger
* Back to your dashboard, go to Jetpack > XML-RPC Errors
* Confirm there's an error with error code `no_possible_tokens` and message `No blog token found` 
* Clear all errors
* go back to Jetpack > broken token and click "Restore from stored options"

### CODE REVIEW

File: packages/connection/src/class-manager.php [hunk 1]
```diff
@@ -2139,7 +2139,7 @@ public function get_access_token( $user_id = false, $token_key = false, $suppres
 
 		if ( $user_id ) {
 			if ( ! $user_tokens ) {
-				return $suppress_errors ? false : new \WP_Error( 'no_user_tokens' );
+				return $suppress_errors ? false : new \WP_Error( 'no_user_tokens', 'No user tokens found' );
```
Comment (dereksmart) [inline packages/connection/src/class-manager.php:2142]: Will these ever be displayed to end users? (do they need to be translated?)

Comment (leogermani) [reply packages/connection/src/class-manager.php:2142]: They are not currently displayed anywhere at all. We want to start displaying them in the Jetpack Debugger now.

I don't think we will use it to display to the user at any point, we will most likely continue what we are doing which is grouping many error codes in one single friendly user message. This detailed error code will serve just for debugging.

In any case, if we decide it's best to translate it, we will have to add i18n function in all error messages there. I just added messages to a few errors that didn't have one, but most of them have - and all of them are not internationalized.

Comment (dereksmart) [inline packages/connection/src/class-manager.php:2142]: It's really cheap to wrap strings in translations, even if they're not shown right now they might be later.

Comment (leogermani) [reply packages/connection/src/class-manager.php]: done! I also added i18n to all the error messages in that function

**Pull request merged**